specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/locations.json

```diff
@@ -123,6 +162,169 @@
+            "type": "string"
+          }
+        },
+        "nodeTypes": {
+          "description": "The list of node types affected by the filter.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "clusterVersions": {
+          "description": "The list of cluster versions affected in Major.Minor format.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "osType": {
+          "description": "The OSType affected, Windows or Linux.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "Windows",
+              "Linux"
+            ],
+            "x-ms-enum": {
+              "name": "OSType",
+              "modelAsString": false
+            }
+          }
+        },
+        "vmSizes": {
+          "description": "The list of virtual machine sizes to include or exclude.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "BillingResources": {
+      "type": "object",
+      "description": "The billing resources.",
+      "properties": {
+        "region": {
+          "description": "The region or location.",
+          "type": "string"
+        },
+        "billingMeters": {
+          "description": "The billing meter information.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BillingMeters"
+          }
+        },
+        "diskBillingMeters": {
+          "description": "The managed disk billing information.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DiskBillingMeters"
+          }
+        }
+      }
+    },
+    "BillingMeters": {
+      "type": "object",
+      "description": "The billing meters.",
+      "properties": {
+        "meterParameter": {
+          "description": "The virtual machine sizes.",
+          "type": "string"
+        },
+        "meter": {
+          "description": "The HDInsight meter guid.",
+          "type": "string"
+        },
+        "unit": {
+          "description": "The unit of meter, VMHours or CoreHours.",
+          "type": "string"
+        }
+      }
+    },
+    "DiskBillingMeters": {
+      "type": "object",
+      "description": "The disk billing meters.",
+      "properties": {
+        "diskRpMeter": {
+          "description": "The managed disk meter guid.",
+          "type": "string"
+        },
+        "sku": {
+          "description": "The managed disk billing sku, P30 or S30.",
+          "type": "string"
+        },
+        "tier": {
+          "description": "The managed disk billing tier, Standard or Premium.",
+          "type": "string",
+          "enum": [
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "Tier",
+            "modelAsString": false
```

Comment:
I think this should be "modelAsString" : true
@ravbhatnagar, do you agree?

Reply:
sure

Reply:
We cannot do this because our SDK is GA now and it will be a breaking change. We will fix the issue in the next API version. Created a GitHub issue #6505  to track this.